### PR TITLE
Fix search suggestions display issue

### PR DESCRIPTION
**Pull request by wikumChamith**

### CODE REVIEW

**Comment by dkayiwa** [comment]
Did you test and confirm that this fixes the reported problem?

**Comment by dkayiwa** [comment]
Can you also fix the `null` in the person's name? For instance `John null Smith` instead of just `John Smith` as it used to be before.

File: omod/src/main/webapp/resources/scripts/jquery/autocomplete/OpenmrsAutoComplete.js [hunk 1]
```diff
@@ -250,11 +250,8 @@ function CreateCallback(options) {
 		
 		textShown = imageText + textShown + ageText; // space was inserted into beginning of 'textShown' var
 
-		textShown = $j('<div/>').text(textShown).html()
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/jquery/autocomplete/OpenmrsAutoComplete.js:253]: Is this intentionally reverting the change made here? https://github.com/openmrs/openmrs-module-legacyui/commit/fd7227cc412724dfe7dcb58bc37ad75921debfc5#diff-ba564bbe7692a2facf544e2b0de8450169b3467afee7aaf1f3b69c2e6e407725L247

**Comment by wikumChamith** [comment]
@dkayiwa I updated the PR.

File: omod/src/main/webapp/resources/scripts/jquery/autocomplete/OpenmrsAutoComplete.js [hunk 2]
```diff
@@ -231,30 +231,40 @@ function CreateCallback(options) {
...
-		
+		if (person.identifier) {
+			textShown += $j('<div/>').text(person.identifier).html();
+		}
+
 		textShown += " ";
 		
-		textShown += person.personName;
+		var personName = "";
+		if (person.givenName !== "null") {
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/jquery/autocomplete/OpenmrsAutoComplete.js:241]: Why put null in double quotes?

Comment (wikumChamith) [reply omod/src/main/webapp/resources/scripts/jquery/autocomplete/OpenmrsAutoComplete.js:241]: The Person object currently stores these values as strings
```
{
    "address1": null,
    "address2": null,
    "age": 0,
    "attributes": {},
    "birthdate": "2024-07-01T18:30:00.000Z",
    "birthdateEstimated": false,
    "birthdateString": "Jul 2, 2024",
    "deathDateString": null,
    "deathdateEstimated": false,
    "familyName": "Doe",
    "gender": "M",
    "givenName": "John",
    "identifier": "1aaa",
    "identifierTypeName": "",
    "middleName": "null",
    "otherIdentifiers": "",
    "otherNames": "",
    "patientId": 4,
    "personId": 4,
    "personName": "John null Doe",
    "uuid": "152075b7-b5b0-4b6f-b9de-0d07b4099afb",
    "voided": false
}
```

**Pull request merged**